org.uqbar.project.wollok.ui.launch/src/org/uqbar/project/wollok/ui/tests/model/WollokTestState.java

```diff
@@ -47,7 +47,7 @@ public URI getURI(WollokTestResult result) {
 
 		@Override
 		public String getOutputText(WollokTestResult result) {
-			return getText();
+			return getText() + " - " + result.getTotalTimeElapsed();
```

Comment:
Renombrando result a it podría ser 

```suggestion
			"""«text»-«totalTimeElapsed»"""
```

Reply:
El tema es que es un `enum` y está escrito en Java, intuyo que porque los enums de Xtend no se la bancan tanto. Quizás en algún momento se pueda migrar a Xtend...